addons/folo/jaxrs/src/main/java/org/commonjava/indy/folo/bind/jaxrs/FoloAdminResource.java

```diff
@@ -138,6 +145,9 @@ public Response recalculateRecord( @ApiParam( "User-assigned tracking session ke
     @Produces( application_zip )
     public File getZipRepository( @ApiParam( "User-assigned tracking session key" ) @PathParam( "id" ) String id )
     {
+        if ( !config.isAdminEnabled() ) {
+            return null;
```

Comment:
I don't think return null is a good idea. We can retrun 404 with a content of "This feature has been disabled. Please contact administrator for details."

Reply:
it;s removed now.